go.mod

```diff
@@ -1,7 +1,9 @@
 module github.com/cilium/cilium
 
 // renovate: datasource=golang-version depName=go
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.10
```

Comment:
These lines look suspicious, it should probably be reverted.

Reply:
reverted an dependency update on cilium/proxy v1.30, now this is gone.